CMSIS/Core/Include/m-profile/cmsis_iccarm_m.h

```diff
@@ -77,7 +77,7 @@
 
 /* Alternativ core deduction for older ICCARM's */
 #if !defined(__ARM_ARCH_6M__) && !defined(__ARM_ARCH_7M__) && !defined(__ARM_ARCH_7EM__) && \
-    !defined(__ARM_ARCH_8M_BASE__) && !defined(__ARM_ARCH_8M_MAIN__)
```

Comment:
Now, as we detect this define is not given, shall we define it below?

Reply:
yes, added